### PR TITLE
updates to mongo lesson

### PR DESCRIPTION
**Pull request by andrewsunglaekim**

### CODE REVIEW

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 1]
```diff
@@ -153,48 +154,33 @@ ThinkShare (2min):
 - Try it
 
 ## What jumped out to me
-- `show x``: show database names
+- `show dbs`: show database names
```
Comment (adambray) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: Nice

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 2]
```diff
@@ -153,48 +154,33 @@ ThinkShare (2min):
...
 ```
-use milk-n-cookies
+use restaurant_db
 ```
 
 Verify:
 ```
 > db
-milk-n-cookies
+restaurant_db
```
Comment (adambray) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: Agreed let's keep it simple domain wise

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 3]
```diff
@@ -480,129 +435,464 @@ kitty.save(function (err) {
...
+In `controllers/authorsController.js`:
+
+```js
+var AuthorModel = require("../models/author")
+var ReminderModel = require("../models/reminder")
+
+var authorsController = {
+  index: function(req, res){
+    AuthorModel.find({}, function(err, docs){
+      res.render("authors/index", {authors: docs})
```
Comment (adambray) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: Maybe use results here instead of docs

Comment (adambray) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: forget this

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 4]
```diff
@@ -374,19 +326,22 @@ http://docs.mongodb.org/manual/core/write-operations-introduction/
...
+
+Update a restaurants to have a new key-value par `{state: "DC"}`
 
 ```
 > db.restaurants.update(
-  {"address.zipcode": 20001},
-  { $set: { state: "DC" }},
-  {multi: true }
+  {"name": "Cookies Corner"},
+  { $set: { state: "DC" }}
```
Comment (mattscilipoti) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: I feel that limiting this change is important.  It is so rare that we want to update *every* document.  But, you skipped the entire discussion of finding (and filtering) documents.

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 5]
```diff
@@ -374,19 +326,22 @@ http://docs.mongodb.org/manual/core/write-operations-introduction/
 )
 ```
 
-### You do (10 min): Update all restaurants in 20001 to be in DC.
+### You do (10 min):
+
+Update a restaurants to have a new key-value par `{state: "DC"}`
```
Comment (mattscilipoti) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: I recommend you pose the problem in English and then discuss the technical solution.

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 6]
```diff
@@ -480,129 +435,463 @@ kitty.save(function (err) {
...
-``` js
-var mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/milk-n-cookies');
+In `models/reminder.js`:
 
-var db = mongoose.connection;
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function (callback) {
-  console.log("Connection established to: ", db.name)
-});
```
Comment (mattscilipoti) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: You removed that message that indicates we have a connection?  Are we so focused on the application that we have lost sight of the mongoose lesson?  As I continue reviewing, I see an awful lot of code that has nothing to do with mongo/mongoose.  To be clear, I'm not expecting it to be changed tonight.  Let's just keep an eye on this for future updates.

File: 08-single-page-web-apps/mongo_mongoose/readme.md [hunk 7]
```diff
@@ -480,129 +435,463 @@ kitty.save(function (err) {
...
+> This is an alternate sytax `.findByIdAndUpdate()`. We can actually us this method and pass in an options object. In this case, we're setting a key of `$pull`. This will find the reminder by the id specified in the url(`req.params.id`) and get rid of it from the author. Then upon success of `.findByIdAndUpdate()` it redirects tot hat author's show page.
+
+## Authors Edit/Update/destroy
+Using what you've learned in class, set up edit update and destroy for authors.
 
-[X] 1. Define the Schema
-[X] 2. Instantiate a model.
-[ ] 3. Find restaurants.
-[ ] 4. Just list names.
-[ ] 5. List names & zipcodes (w/o _id)
```
Comment (mattscilipoti) [inline 08-single-page-web-apps/mongo_mongoose/readme.md]: Looks like you have removed all the lists that periodically reminded us of the overview and the steps we were taking.  With all this code, I feel we need to work harder on keeping our attention focused on the steps.  On the reasons why we are writing all this code.

**Pull request merged**